index.html

```diff
@@ -140,76 +190,254 @@ <h2>
+          is a <a data-cite="service-workers">service worker client</a>, and
+          returns a <a data-cite="infra#ordered-set">set</a> of [=web
+          applications=]:
+        </p>
+        <ol>
+          <li>If [=current settings object=] has a [=environment settings
+          object/responsible document=] |document:Document|:
+            <ol>
+              <li>Let |apps:set of applications| be the <a data-cite=
+              "infra#ordered-set">set</a> of installed [=web applications=]
+              such that |document|'s <a data-cite=
+              "dom#concept-document-url">URL</a> is <a data-cite=
+              "appmanifest#dfn-within-scope-manifest">within scope</a> of the
+              application's manifest. (Order is not important.)
+              </li>
+              <li>[=list/Remove=] all elements of |apps| other than the one
+              with the longest (i.e., most specific) [=navigation scope=].
+              </li>
+              <li>Return |apps|.
+              </li>
+            </ol>
+          </li>
+          <li>Otherwise:
+            <ol>
+              <li>Let |worker:service worker client| be the [=current settings
+              object=] (assumed to be a <a data-cite="service-workers">service
+              worker client</a>).
+              </li>
+              <li>Return the <a data-cite="infra#ordered-set">set</a> of
+              installed [=web applications=] whose [=navigation scope=] is
+              <a data-cite="appmanifest#dfn-within-scope-manifest">within
+              scope</a> (as defined in [[AppManifest]]) of |worker|'s
+              <a data-cite=
+              "service-workers-1#dfn-containing-service-worker-registration">containing
+              service worker registration</a>'s <a data-cite=
+              "service-workers-1#dfn-scope-url">scope url</a>. (Order is not
+              important.)
+              </li>
+            </ol>
+          </li>
+        </ol>
+      </section>
+    </section>
+    <section class="informative">
+      <h2>
+        Security and privacy considerations
+      </h2>
+      <p>
+        The API is write-only by design. There is no way for a site to read
+        back the value of a badge that was previously set, to ensure that the
+        application badge cannot be used as a storage or fingerprinting
+        mechanism.
+      </p>
+      <div class="issue" data-number="44"></div>
+    </section>
+    <section class="informative">
+      <h2>
+        Accessibility considerations
+      </h2>
+      <div class="issue" data-number="24"></div>
     </section>
     <section>
       <h2>
         Dependencies
       </h2>
       <p>
-        <code><dfn data-cite=
-        "!ECMASCRIPT#sec-native-error-types-used-in-this-standard-typeerror">TypeError</dfn></code>
-        is defined by [[ECMASCRIPT]].
+        <dfn data-cite="appmanifest#installable-web-applications" data-lt=
```

Comment:
Could you please file a bug on Manifest to export these out? Then we can get rid of this 🎉

Reply:
Sent https://github.com/w3c/manifest/pull/823